website/server/controllers/api-v3/user.js

```diff
@@ -1310,4 +1310,28 @@ api.userReset = {
   },
 };
 
+/**
+* @api {post} /api/v3/user/set-custom-day-start Sets preferences.dayStart for user
+* @apiVersion 3.0.0
+* @apiName UserSetCustomDayStart
+* @apiGroup User
+*
+* @apiSuccess {Object} data An empty Object
+*/
+api.userSetCustomDayStart = {
+  method: 'POST',
+  middlewares: [authWithHeaders()],
+  url: '/user/set-custom-day-start',
+  async handler (req, res) {
+    let user = res.locals.user;
+    let dayStart = req.body.dayStart;
```

Comment:
Again, I was just mimicking the debug route. I can put an is empty check on this. We seemed to want some other validation, but I didn't see clarification.